features/support/examples/rspec/spec_helper_only_rack_test.rb.example

```diff
@@ -3,7 +3,6 @@ require "rspec/core"
 require "rspec/matchers"
 require "rspec/expectations"
 require "raml-rb"
-require "rack/test"
 
 path = File.expand_path('../../lib', __FILE__)
 
```

Comment:
I think that makes sense. So I think what we should do here is not add requires for Rack::Test and JSON if they aren't already there, but create a spec helper with those requires if there is no spec helper already. Then, we can delete the files we're using to test the adding-stuff-to-an-existing-spec_helper behavior. Sound reasonable?